test/common/ValueGuesser.ts

```diff
@@ -15,9 +15,9 @@ function check(values: Array<string | null>, expectedResult: GuessResult) {
 describe("ValueGuesser", function() {
   it("should guess booleans and numbers correctly", function() {
     check(
-      ["true", "false"],
+      ["true", "false", "yes", "no"],
       {
-        values: [true, false],
+        values: [true, false, true, false],
```

Comment:
Would it be relevant to add test to be sure  that `"1"` is truthy and `"0"` is falsy?